sql/core/src/test/scala/org/apache/spark/sql/errors/QueryErrorsSuiteBase.scala

```diff
@@ -51,4 +51,17 @@ trait QueryErrorsSuiteBase extends SharedSparkSession {
     assert(exception.getSqlState === sqlState)
     assert(exception.getMessage === s"""\n[$fullErrorClass] """ + message)
   }
+
+  case class ExpectedContext(
```

Comment:
why do we need this? can we use `SQLQueryContext` directly in the test?

Reply:
For simplicity, `ExpectedContext` contains only import info provided via `QueryContext`. For instance, SQLQueryContext contains entire SQL text NOT fragment. Why should we check entire SQL statements in tests?